### DISCUSSION TITLE
[QUESTION] Why is IM6 & IM7 not in benchmarks anymore?

### DISCUSSION THREAD

**Discussion by the-hotmann** [libvips/libvips#3015] [Q&A]
Sorry to abuse this Bug report limited Issue section, but I noticed, that since [17 Jul 2019](https://github.com/libvips/libvips/wiki/Speed-and-memory-use/376dbaa833c060f1ad87ec47450cca9466af75dc) no new Benchmarks of ImageMagick 6 & 7 have been posted.

The next version of the this page [950ffe3](https://github.com/libvips/libvips/wiki/Speed-and-memory-use/950ffe3bd7c1a3e6a9cdd7a0902de5c5ef652fe5) does not include the benchmarks anymore. So comparing them will be harder. I am really keen to know how libvips compared to IM6 & IM7 performs and I think a lot other people would love to know aswell.

I am open for a discussion and ofc also for having this "issues" closed down without answer, as it is not a real bug.

Best regards.

**Comment by jcupitt** [comment]
Hi @MartinHotmann,

It's true I've not updated it for a while, I think June 2021 was the last time:

https://github.com/libvips/libvips/wiki/Speed-and-memory-use

I'll rerun the tests on ubuntu 22.04. I forget why I omitted im7, I'll add it back.

**Reply by jcupitt** [reply]
Though you can expect im7 to be about half the speed of im6. It uses floats for pixels (im6 defaults to int16), so images are twice the size in memory. Twice the size is half the speed (very roughly).

**Comment by jcupitt** [comment]
Now I remember why I hadn't updated the tests -- there was a small performance regression when I last looked that needs chasing down first. I'll look into that again.
